cp3-networkpolicy/egress/zen/zen-egress-zen-remote-svc-inst-status-cron-job.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    component: cpfs
```

```suggestion
    component: cpfs3
```

Hi @psnavale 
Can we update the label to `component: cpfs3`? All others look good to me! Thank you!

---

ok sure